DigitalLearningSolutions.Web/Helpers/RegistrationMappingHelper.cs

```diff
@@ -8,33 +8,86 @@ public static class RegistrationMappingHelper
-                data.LearnerInformationViewModel.Answer6
+                data.Answer1,
+                data.Answer2,
+                data.Answer3,
+                data.Answer4,
+                data.Answer5,
+                data.Answer6
             );
         }
 
-        public static SummaryViewModel MapToSummary(DelegateRegistrationData data, string centre, string jobGroup)
+        public static SummaryViewModel MapDataToSummary(DelegateRegistrationData data)
         {
             return new SummaryViewModel
-            (
-                data.RegisterViewModel.FirstName!,
-                data.RegisterViewModel.LastName!,
-                data.RegisterViewModel.Email!,
-                centre,
-                jobGroup
-            );
+            {
+                FirstName = data.FirstName!,
+                LastName = data.LastName!,
+                Email = data.Email!,
+                IsCentreSpecificRegistration = data.IsCentreSpecificRegistration
+            };
+        }
+
+        public static PersonalInformationViewModel MapDataToPersonalInformation(DelegateRegistrationData data)
+        {
+            return new PersonalInformationViewModel
+            {
+                FirstName = data.FirstName,
+                LastName = data.LastName,
+                Centre = data.Centre,
+                Email = data.Email,
+                IsCentreSpecificRegistration = data.IsCentreSpecificRegistration
+            };
+        }
+
+        public static LearnerInformationViewModel MapDataToLearnerInformation(
+            DelegateRegistrationData data
+        )
+        {
+            return new LearnerInformationViewModel
+            {
+                JobGroup = data.JobGroup,
+                Answer1 = data.Answer1,
+                Answer2 = data.Answer2,
+                Answer3 = data.Answer3,
+                Answer4 = data.Answer4,
+                Answer5 = data.Answer5,
+                Answer6 = data.Answer6
+            };
+        }
+
+        public static DelegateRegistrationData MapPersonalInformationToData(
+            PersonalInformationViewModel model,
+            DelegateRegistrationData data
+        )
+        {
+            data.Centre = model.Centre;
+            data.Email = model.Email;
+            data.FirstName = model.FirstName;
+            data.LastName = model.LastName;
+            return data;
+        }
+
+        public static DelegateRegistrationData MapLearnerInformationToData(
+            LearnerInformationViewModel model,
+            DelegateRegistrationData data
+        )
+        {
+            data.JobGroup = model.JobGroup;
+            data.Answer1 = model.Answer1;
+            data.Answer2 = model.Answer2;
+            data.Answer3 = model.Answer3;
+            data.Answer4 = model.Answer4;
+            data.Answer5 = model.Answer5;
+            data.Answer6 = model.Answer6;
+            return data;
         }
     }

```

Comment:
Using the mapper to get ViewModel is great (I think we could do some of it in the ViewModels' constructors but David mentioned that there was some weird behaviour around this back in the day) 

I wonder if instead of the ViewModel to Data methods, we could declare some public methods on the data object - e.g. SetPersonalInformation(model), SetLearnerInformation(model), ClearCustomPromptAnswers() (we can just move the relevant tests to view model tests).